.github/workflows/fuzzing.yml

```diff
@@ -79,3 +79,11 @@ jobs:
           name: ${{ matrix.sanitizer }}-artifacts
           retention-days: 21
           path: ./out/artifacts
+      - name: Sync artifacts
```

Comment:
For now, we don't need to do that for this job.

Reply:
fixed